### DISCUSSION TITLE
IBM i Connection issue

### DISCUSSION THREAD

**Discussion by Mef08051015** [codefori/vscode-ibmi#729] [IBM i] [answered]
I'm trying to connect to a Client's IBM i and get the error message "Not Connected xxx.xxx.xxx.xxx All configured Authentication Methods Failed"  My User Id is part of the QPGMR  Group and AllowUserGroups qpgmr is in the sshd_config. I have stopped and restarted the sshd server. I can connect with a User Profile with All Object Authority. 

What authorities are required to connect? I have successfully connected to my IBM i using a QPGMR Group User Id with the same authorities as my Client's System.

**Comment by worksofliam** [comment]
@Mef08051015 the next best thing to try is to try connecting using ssh from the command line instead. That will help you/us diagnose issues faster.

**Comment by Mef08051015** [comment]
Here's the result from the command line:


![SSH Connection 1](https://user-images.githubusercontent.com/108477685/176763048-c13e5123-486f-4dbf-b8a6-08747ef5cce8.jpg)

**Reply by worksofliam** [reply]
You need to use ssh on your local machine, not the IBM i.

**Comment by andreas-prouza** [comment, marked as answer]
There are some requirements to be able to start a SSH session.

A valid Shell is set
If you set your profile to Bash but Bash doesn't exist, then you are also not able to login
CALL QSYS2.SET_PASE_SHELL_INFO('*CURRENT', '/QOpenSys/pkgs/bin/bash')
You should be the owner of your home directory
Only you should have write permission to your home directory

**Reply by Mef08051015** [reply]
Thank you so much Andreas, that was it. After setting the Shell and adding AlloweUsers to the ssd_config file for myself and the other programmers, everything works perfectly.
